### DISCUSSION TITLE
Close all PDF

### DISCUSSION THREAD

**Discussion by mayank265** [sumatrapdfreader/sumatrapdf#2471] [Ideas]
I opened 15 pdf in Sumatra. Now I have to close them individually.
There should be these options like web browser
Right Clicking a tab --> Close Tab To Right, Close Tab To Left, Close All Tab, Close All Tab but This.

**Comment by GitHubRulesOK** [comment]
The close all tabs can be workaround emulated using CTRL + N then close old window. likewise using CTRL+SHIFT+N will open current in new window and you can close ALL except by closing the first window.

Agreed its a workaround until someone changes all the relevant tabs coding, as not a 5 seconds job.

**Comment by kjk** [comment]
We have a request in issues: https://github.com/sumatrapdfreader/sumatrapdf/issues/2288

I'm working on a Command Palette and when that's done then I would likely implement such thing via command palette.

**Comment by Nailuj1** [comment]
Closing with mouse-wheel click is kind of annoying cause you first have to click on the specific tab, then you are able with another click to close it. So if you wanna close some pdfs you have to click a lot ;) 
I think a solution would be really simple cause the under-mouse-tab is already highlighted.
